chocolatey/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,39 @@
+﻿$ErrorActionPreference = 'Stop'; # stop on all errors
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  softwareName  = 'nvs'  #part or all of the Display Name as you see it in Programs and Features. It should be enough to be unique
+  fileType      = 'msi' #only one of these: MSI or EXE (ignore MSU for now)
+  # MSI
+  silentArgs    = "/qn /norestart"
+  validExitCodes= @(0, 3010, 1605, 1614, 1641) # https://msdn.microsoft.com/en-us/library/aa376931(v=vs.85).aspx
+}
+
+$uninstalled = $false

```

Comment:
This `uninstalled` variable doesn't appear to do anything.

Reply:
I know, it came with the autogenerated scripts so I wasn't sure about removing it because maybe it's used somewhere else in the chocolatey process. I'll try removing it see what happens.